bundle/witbundle/bundle.go

```diff
@@ -0,0 +1,202 @@
+package witbundle
```

Comment:
@jsnctl I wonder if we should keep everything WIT-SVID related in an `exp` folder to indicate clearly that this is still experimental? Otherwise there's no differentiation between X509, JWT and WIT SVID and users of this SDK get the perception that this is fully supported.

Once the RFC is finished and has a number we can then move it into the main part of the SDK.

Reply:
I've added the new `exp` package, and added a brief `doc.go` to it https://github.com/jsnctl/go-spiffe/blob/jsnctl/wit-bundle/exp/doc.go